### PR TITLE
Adjust allowed bot list to allow for link previews in WhatsApp and Signal

### LINKED ISSUE #1392: Add Signal's user-agent to our bot list
So link previews can be rendered properly.

image

### PR DESCRIPTION
**Pull request by lindner**
See signalapp/Signal-Android#10060 for how Signal uses their UA

Fixes #1392

### CODE REVIEW

**Comment by gabek** [comment]
hahahahaha oh man.  Thanks for looking into that.  I never would have expected that one.

File: utils/utils_test.go [hunk 1]
```diff
@@ -7,6 +7,7 @@ func TestUserAgent(t *testing.T) {
 		"Pleroma 1.0.0-1168-ge18c7866-pleroma-dot-site; https://pleroma.site info@pleroma.site",
 		"Mastodon 1.2.3 Bot",
 		"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_5) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/13.1.1 Safari/605.1.15 (Applebot/0.1; +http://www.apple.com/go/applebot)",
+                "WhatsApp",
```
Comment (gabek) [inline utils/utils_test.go]: Looks like some weird spacing snuck in here.

Comment (lindner) [reply utils/utils_test.go]: Fixed by running gofmt (oops!)

**Comment by gabek** [comment]
Thanks again!

**Pull request merged**